oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/web/OAuth2AuthorizationCodeGrantFilterTests.java

```diff
@@ -130,6 +130,12 @@ public void setAuthorizationRequestRepositoryWhenAuthorizationRequestRepositoryI
 				.isInstanceOf(IllegalArgumentException.class);
 	}
 
+	@Test
```

Comment:
We should also have a test that asserts the supplied `RequestCache` is actually used.